src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+
+    private static final FigureSupplier figureSupplies = new FigureSupplier();
+
+    public static void main(String[] args) {
+
+        Figure getRandomFIgure = figureSupplies.getRandomFigure();
+        Figure getDefaultFigure = figureSupplies.getDefaultFigure();
+
+        Figure[] array = {getRandomFIgure, getDefaultFigure};
+
+        for (Figure figures : array) {
```

Comment:
The variable name `figures` in the for-each loop suggests multiple figures, but it represents a single figure. Consider renaming it to `figure` for clarity.